code/_onclick/hud/screen_objects.dm

```diff
@@ -678,9 +678,10 @@
 	..()
 	holder = C
 	holder.screen += src
-	var/titlescreen = TITLESCREEN
-	if(titlescreen)
-		icon_state = titlescreen
+
+	if(SStitle.title_screen)
+		icon_state = SStitle.title_screen.icon_state

```

Comment:
I don't like that you're depending on the splashscreen turfs existing. Store the icon_state in SStitle